onnxruntime/hosting/server_configuration.h

```diff
@@ -82,6 +95,15 @@ class ServerConfiguration {
     return vm.count("help") || vm.count("h");
   }
 
+  onnxruntime::logging::Severity GetSeverity(const std::string& level) {
+    if (level == "verbose") return onnxruntime::logging::Severity::kVERBOSE;
```

Comment:
Can we use an `unordered_map` here?

Reply:
As we said offline there's no reason to do this